guides/installation/valet.md

```diff
@@ -8,49 +8,54 @@ This is a modified version of the [official Installation Guide](https://github.c
 
 ## If you have Valet installed
 
 Run `composer remove laravel/valet`.
 
 ## Installing Valet-PHP
 
-1. Update Homebrew via `brew update`
-1. Add the Homebrew PHP tap for Valet+ via `brew tap henkrehorst/php`
-1. Install PHP 7.4 using Homebrew via `brew install valet-php@7.4`
-1. Link your PHP version using the `brew link valet-php@7.4 --force` command
+- Update Homebrew via `brew update`.
+- Add the Homebrew PHP tap for Valet+ via `brew tap henkrehorst/php`.
+- Install PHP 7.4 using Homebrew via `brew install valet-php@7.4`.
+- Link your PHP version using the `brew link valet-php@7.4 --force` command.
 
 ## Installing Valet+
 
-1. If needed, install composer via `brew install composer`
-1. Install Valet+ via `composer global require weprovide/valet-plus`
-1. Make sure `~/.composer/vendor/bin` is in your path by adding `export PATH="$PATH:$HOME/.composer/vendor/bin"` to your `bash_profile` or `.zshrc`
-1. Check for the following, common problem with `valet fix`. **Warning: This will uninstall all other PHP installations**
-1. Run the `valet install` command. Optionally add `--with-mariadb` to use MariaDB instead of MySQL. This will configure and install Valet+ and DnsMasq.
+- If needed, install composer via `brew install composer`.
+- Install Valet+ via `composer global require weprovide/valet-plus`.
+- Make sure `~/.composer/vendor/bin` is in your path by adding `export PATH="$PATH:$HOME/.composer/vendor/bin"` to your `bash_profile` or `.zshrc`.
+- Check for the following common problem with `valet fix`.
+{% hint style="warning" %}
+The above instruction will uninstall all other PHP installations.
+{% endhint %}
+- Run the `valet install` command. Optionally add `--with-mariadb` to use MariaDB instead of MySQL. This will configure and install Valet+ and DnsMasq.
 
-   Additionally, it registers Valet's daemon to launch when your system starts.
+Additionally, it registers Valet's daemon to launch when your system starts.
 
 ## Using Valet+ with Shopware 6
 
-1. Create a new empty folder for example `~/sites`
-1. Clone the development template like you normally would \(dev + platform\) into this folder
-1. Adjust params instalation editing `.psh.yaml.dist`
-1. Run `./psh.phar install`
-1. Move to `~/sites` and run `valet park` to register valet for this directory. Shopware should now be accessible via the `folder-name.test`. Notice: "folder-name" is the name of the Shopware development template in `~/sites`
-1. Optional: Disable SSL via `valet unsecure` because this might cause problems with the watcher
+- Create a new empty folder, for example `~/sites`.
+- Clone the development template like you normally would \(dev + platform\) into this folder.
+- Adjust params installation editing `.psh.yaml.dist`.
+- Run `./psh.phar install`.
+- Move to `~/sites` and run `valet park` to register Valet for this directory. Shopware should now be accessible via the `folder-name.test`.
+{% hint style="warning" %}
```

Comment:
I don't think hints work within lists. Also, the style should be "info" if it's a notice.